.github/workflows/trivy.yml

```
@@ -0,0 +1,25 @@
+  push:
+    branches:
+    - main
+
+permissions:
+  contents: write
+
+jobs:
+  generate-sbom:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)